src/helpers/git_helpers/merge_helpers.sh

```diff
@@ -45,3 +45,13 @@ function commit_squash_merge {
 function undo_steps_for_merge {
   echo "reset_to_sha $(current_sha) hard"
 }
+
+
+function post_undo_steps_for_commit_squash_merge {
+  local current_branch_name=$(get_current_branch_name)
+  echo "checkout $current_branch_name"
```

Comment:
This checkout seems unnecessary since the current branch name appears to be the branch you are currently on